web_m2x_options/static/src/components/form.esm.js

```diff
@@ -56,6 +60,18 @@ Many2ManyTagsFieldColorEditable.props = {
     fieldColorOptions: {type: Object, optional: true},
 };
 
+ListMany2ManyTagsAvatarField.props = {
+    ...ListMany2ManyTagsAvatarField.props,
+    fieldColor: {type: String, optional: true},
+    fieldColorOptions: {type: Object, optional: true},
+};
```

Comment:
Question: is ListMany2ManyTagsAvatarField necessary, because it derives from Many2ManyTagsAvatarField:
export class ListMany2ManyTagsAvatarField extends Many2ManyTagsAvatarField

Reply:
That is a good point, actually I fixed first the ListMany2ManyTagsAvatarField and then noticed that I was still getting the issue with Many2ManyTagsAvatarField so I applied directly the same fix.

I will check by just fixing the latter to make sure we do not add unnecessary code and open a new PR. Thanks for noticing!